### PR TITLE
dont send deployment.dev events to sentry

### PR DESCRIPTION
**Pull request by TomasTomecek**
I was getting annoyed by the nasty tracebacks lately.

### CODE REVIEW

**Comment by csomh** [comment]
@lachmanfrantisek @lbarcziova @dhodovsk @sakalosj was any of you checking these sentry alerts from dev envs? is it okay if disabling it is hardcoded, or better to be  configurable instead?

**Comment by TomasTomecek** [comment]
> @lachmanfrantisek @lbarcziova @dhodovsk @sakalosj was any of you checking these sentry alerts from dev envs? is it okay if disabling it is hardcoded, or better to be configurable instead?

Thanks, we probably should have this discussion before merging. As I wrote in the OP, I got frustrated from seeing ugly errors which are not happening in the official deployments.

Will add this to arch mtg.

File: packit_service/sentry_integration.py [hunk 1]
```diff
@@ -77,10 +78,11 @@ def configure_sentry(
 
 
 def send_to_sentry(ex):
-    # so that we don't have to have sentry sdk installed locally
-    import sentry_sdk
+    if ServiceConfig.get_service_config().deployment != Deployment.dev:
```
Comment (lachmanfrantisek) [inline packit_service/sentry_integration.py:81]: Cannot this be handled in the sentry-setup?

Comment (TomasTomecek) [reply packit_service/sentry_integration.py:81]: Please elaborate.

Comment (lachmanfrantisek) [reply packit_service/sentry_integration.py:81]: We have a place in code where we setup entry:
- If this "filtering" can be moved there.
- Check if sentry itself cannot handle this "filtering".

**Pull request closed without merging**